int64 G_State
int32 G_ID
int64 x
int64 y
int64 lockedBy

